iib/workers/dogpile_cache.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import functools
+
+from dogpile.cache import make_region
+
+from iib.workers.config import get_worker_config
+
+
+def should_use_cache(*args):
```

Comment:
Could you make this a clearer name?
```suggestion
def skopeo_inspect_should_use_cache(*args):
```

Reply:
sure, thanks its better this way